src/pages/instances/InstanceOverview.tsx

```diff
@@ -1,192 +1,168 @@
-                      " disk used"
-                    }
-                  />
-                </div>
-              ) : (
-                "-"
-              )}
-            </td>
-          </tr>
-          <tr>
-            <th className="u-text--muted">Type</th>
-            <td>{instance.type}</td>
-          </tr>
-          <tr>
-            <th className="u-text--muted">Architecture</th>
-            <td>{instance.architecture}</td>
-          </tr>
-          <tr>
-            <th className="u-text--muted">Image</th>
-            <td>{instance.config["image.description"]}</td>
-          </tr>
-          <tr>
-            <th className="u-text--muted">Stateful</th>
-            <td>{instance.stateful ? "true" : "false"}</td>
-          </tr>
-          <tr>
-            <th className="u-text--muted">Project</th>
-            <td>{instance.project}</td>
-          </tr>
-          <tr>
-            <th className="u-text--muted">Profiles</th>
-            <td>
-              <List
-                className="u-no-margin--bottom"
-                items={instance.profiles.map((name) => (
-                  <Link
-                    key={name}
-                    to={`/ui/${instance.project}/profiles/detail/${name}`}
-                  >
-                    {name}
-                  </Link>
-                ))}
-              />
-            </td>
-          </tr>
-          <tr>
-            <th className="u-text--muted">Location</th>
-            <td>{instance.location}</td>
-          </tr>
-          <tr>
-            <th className="u-text--muted">Ephemeral</th>
-            <td>{instance.ephemeral ? "true" : "false"}</td>
-          </tr>
-        </tbody>
-      </table>
-    </>
+    <div className="instance-overview-tab">
+      <NotificationRowLegacy
+        notification={inTabNotification}
+        onDismiss={() => setInTabNotification(null)}
+      />
+      <Row className="general">
+        <Col size={3}>
+          <h2 className="p-heading--4">General</h2>
+        </Col>
+        <Col size={7}>
+          <table>
+            <tbody>
+              <tr>
+                <th className="p-muted-heading">Base image</th>
+                <td>{instance.config["image.description"] ?? "-"}</td>
+              </tr>
+              <tr>
+                <th className="p-muted-heading">Description</th>
+                <td>{instance.description ? instance.description : "-"}</td>
+              </tr>
+              <tr>
+                <th className="p-muted-heading">Type</th>
+                <td>
+                  {
+                    instanceCreationTypes.filter(
+                      (item) => item.value === instance.type
+                    )[0].label
+                  }
+                </td>
+              </tr>
+              <tr>
+                <th className="p-muted-heading">IPv4</th>
+                <td>
+                  {ip4Addresses.length ? (
```

Comment:
This can be a reused component to limit / show all items from an array. We pass it the items and the initial limit. It contains the show all button and holds the state of how many items are currently visible.
We can reuse it here for ipv4 and ipv6, and also use it for both in the detail panel.

We should probably also use it in the Instance list columns for ipv4 and v6 -- but maybe not in scope of this PR.

Reply:
Nice idea! I just implemented that.